### PR TITLE
Ji4 languagesupport

### PR DESCRIPTION
**Pull request by palleBrandt**

### CODE REVIEW

**Review by Maquidos — APPROVED** [review]

File: lib/l10n/app_da.arb
Comment (allanzigge) [inline lib/l10n/app_da.arb:1]: Should description be in danish as well...?

**Review by allanzigge — CHANGES_REQUESTED** [review]
At least for best practice - don't know if it actually affects things

File: lib/l10n/app_da.arb
Comment (palleBrandt) [reply lib/l10n/app_da.arb:1]: Good point, I'll take a look at it later today 👍

Comment (palleBrandt) [reply lib/l10n/app_da.arb:1]: I have looked into this. The description is not for accessibility, but only for developers to know where to use the given key in the application. Therefore I suggest keeping it in English.

Comment (allanzigge) [reply lib/l10n/app_da.arb:1]: *tommel op*

**Review by allanzigge — APPROVED** [review]

**Pull request merged**